nss-day-cohort-47/ldcc-bvinson23
# Bv adminaddtype

# Description
Added functionality to the add a type button but only for admin
Fixes # (issue)
## Type of change
# Testing Instructions
Login in as an admin and add a type. Then login as a non admin and make sure the add a type button does appear
# Checklist:
